CRM/Core/Payment.php

```diff
@@ -682,7 +682,6 @@ public function getPaymentFormFieldsMetadata() {
         'htmlType' => 'text',
         'name' => 'credit_card_number',
         'title' => ts('Card Number'),
-        'cc_field' => TRUE,
```

Comment:
per later removed todo - "// @todo - remove the cc_field check - no longer useful."